scale-tests/configs/2018-08-08-kafka110s-small-scale.env

```diff
@@ -0,0 +1,125 @@
+# Depends on:
+# - TEST_NAME
+# - TEST_S3_BUCKET
+# - TEST_S3_FOLDER
+
+# Infrastructure configuration #################################################
+
+SERVICE_NAMES_PREFIX="${TEST_NAME}/"
+INFRASTRUCTURE_OUTPUT_FILE="${TEST_NAME}-infrastructure.json"
+KAFKA_ZOOKEEPER_CONFIG='scale-tests/configs/kafka-zookeeper-options.json'
+KAFKA_CLUSTER_COUNT=1
+KAFKA_CONFIG='scale-tests/configs/kafka-options.json'
+CASSANDRA_CLUSTER_COUNT=1
+CASSANDRA_CONFIG='scale-tests/configs/cassandra-options.json'
+
+# Non-GPU dispatchers configuration ############################################
+
+NON_GPU_NUM_DISPATCHERS=10
+NON_GPU_DISPATCHERS_OUTPUT_FILE="${TEST_NAME}-non-gpu-dispatchers-${NON_GPU_NUM_DISPATCHERS}.out"
+NON_GPU_DISPATCHERS_JSON_OUTPUT_FILE="${NON_GPU_DISPATCHERS_OUTPUT_FILE}-dispatchers.json" # NOTE: this name is set internally by the deploy dispatchers script
+NON_GPU_QUOTA_DRIVERS_CPUS=20
+NON_GPU_QUOTA_DRIVERS_MEM=50000
+NON_GPU_QUOTA_EXECUTORS_CPUS=25
+NON_GPU_QUOTA_EXECUTORS_MEM=40000
+
+# GPU dispatchers configuration ################################################
+
+GPU_NUM_DISPATCHERS=2
+GPU_DISPATCHERS_OUTPUT_FILE="${TEST_NAME}-gpu-dispatchers-${GPU_NUM_DISPATCHERS}.out"
+GPU_DISPATCHERS_JSON_OUTPUT_FILE="${GPU_DISPATCHERS_OUTPUT_FILE}-dispatchers.json" # NOTE: this name is set internally by the deploy dispatchers script
+GPU_QUOTA_DRIVERS_CPUS=16
+GPU_QUOTA_DRIVERS_MEM=200000
+GPU_QUOTA_EXECUTORS_CPUS=
+GPU_QUOTA_EXECUTORS_MEM=
+GPU_QUOTA_EXECUTORS_GPUS=
+# NOTE: to test Core team’s hypothesis of GPU quota negatively impacting spark
+# launch rates, we remove the GPU quota entirely from all executor roles.
+GPU_REMOVE_EXECUTORS_ROLES_QUOTAS=true
+
+# Common streaming jobs configuration ##########################################
+
+TEST_ASSEMBLY_JAR_URL='http://infinity-artifacts.s3.amazonaws.com/scale-tests/dcos-spark-scala-tests-assembly-20180612-5fa9420.jar'
+NUM_DISPATCHERS="$((${NON_GPU_NUM_DISPATCHERS} + ${GPU_NUM_DISPATCHERS}))"
+DISPATCHERS_JSON_OUTPUT_FILE="${TEST_NAME}-all-dispatchers-${NUM_DISPATCHERS}.json"
+
+# Failing streaming jobs configuration #########################################
+
+FAILING_SUBMISSIONS_OUTPUT_FILE="${TEST_NAME}-failing-submissions.out"
+FAILING_NUM_PRODUCERS_PER_KAFKA="${NUM_DISPATCHERS}"
+FAILING_NUM_CONSUMERS_PER_PRODUCER=1
+FAILING_PRODUCER_NUMBER_OF_WORDS=1300000
+FAILING_PRODUCER_WORDS_PER_SECOND=1
```

Comment:
FYI the failing streaming jobs is no longer a requirement and we haven't run this portion of the script in at least the last 3 scale tests. We could probably remove this altogether but that's for another day.

Reply:
Let's remove this whenever we also remove the script section that uses these variables.

Reply:
Why are the failing jobs not longer a requirement? Was this not one of the things that placed Mesos under load and indicated that DRF was not giving Spark offers?

Reply:
This was originally an accident (config was set incorrectly resulting in continuously failing streaming jobs). Around 3 or 4 scale tests ago, I believe @imaxxs and @triclambert agreed that this portion of the test was not necessary as it isn't testing a realistic load.